InjectionBundle/SwiftInjection.swift

```diff
@@ -314,8 +314,13 @@ extension NSObject {
             if className.hasPrefix("_") {
                 return
             }
+
+            if className.starts(with: "NS") {
+                return
+            }
+
             #if os(OSX)
-            if className.starts(with: "NS") && cls != NSWindow.self {
+            if cls != NSWindow.self {
                 return
             }
```

Comment:
Don’t think this will ever be true now, unsure if that is an issue. What is the idea behind only allowing NSWindow here? Is it to get the initial class ?